<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>UIconfiguracion</class>
  <widget class="QDialog" name="UIconfiguracion" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>379</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Configuración</string>
    </property>
    <layout class="QVBoxLayout" >
      <item>
        <widget class="QTabWidget" name="m_tab" >
          <widget class="QWidget" name="tab" >
            <attribute name="title" >
              <string>B&amp;ulmaGés</string>
            </attribute>
            <layout class="QVBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <item>
                <layout class="QHBoxLayout" >
                  <property name="margin" >
                    <number>0</number>
                  </property>
                  <item>
                    <widget class="QLineEdit" name="lineEditA_1" >
                      <property name="enabled" >
                        <bool>true</bool>
                      </property>
                      <property name="text" >
                        <string>Nom Empresa</string>
                      </property>
                      <property name="readOnly" >
                        <bool>true</bool>
                      </property>
                      <property name="toolTip" stdset="0" >
                        <string>Nombre de la empresa</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <spacer name="spacer2" >
                      <property name="sizeHint" >
                        <size>
                          <width>211</width>
                          <height>21</height>
                        </size>
                      </property>
                      <property name="sizeType" >
                        <enum>Expanding</enum>
                      </property>
                      <property name="orientation" >
                        <enum>Horizontal</enum>
                      </property>
                    </spacer>
                  </item>
                </layout>
              </item>
              <item>
                <layout class="QHBoxLayout" >
                  <property name="margin" >
                    <number>0</number>
                  </property>
                  <item>
                    <widget class="QLineEdit" name="m_ejercicio" >
                      <property name="sizePolicy" >
                        <sizepolicy>
                          <hsizetype>0</hsizetype>
                          <vsizetype>0</vsizetype>
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                        </sizepolicy>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QPushButton" name="pushButtonA_5" >
                      <property name="text" >
                        <string>&amp;Nuevo Ejercicio</string>
                      </property>
                      <property name="toolTip" stdset="0" >
                        <string>Crea un nuevo ejercicio contable</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <spacer name="spacer10" >
                      <property name="sizeHint" >
                        <size>
                          <width>201</width>
                          <height>21</height>
                        </size>
                      </property>
                      <property name="sizeType" >
                        <enum>Expanding</enum>
                      </property>
                      <property name="orientation" >
                        <enum>Horizontal</enum>
                      </property>
                    </spacer>
                  </item>
                </layout>
              </item>
              <item>
                <layout class="QHBoxLayout" >
                  <property name="margin" >
                    <number>0</number>
                  </property>
                  <item>
                    <widget class="QPushButton" name="m_backup" >
                      <property name="text" >
                        <string>Hacer copia &amp;de Seguridad
 (BACKUP)</string>
                      </property>
                      <property name="toolTip" stdset="0" >
                        <string>Guarda una copia de seguridad de una empresa</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QPushButton" name="pushButtonA_4" >
                      <property name="text" >
                        <string>Res&amp;taurar copia
 (RESTORE)</string>
                      </property>
                      <property name="toolTip" stdset="0" >
                        <string>Restaura la copia de seguridad de una empresa</string>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
              <item>
                <layout class="QHBoxLayout" >
                  <property name="margin" >
                    <number>0</number>
                  </property>
                  <item>
                    <widget class="QPushButton" name="pushButtonA_1" >
                      <property name="text" >
                        <string>&amp;Crear Nueva Empresa</string>
                      </property>
                      <property name="toolTip" stdset="0" >
                        <string>Crea una empresa nueva sin datos</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QPushButton" name="m_borrarempresa" >
                      <property name="text" >
                        <string>&amp;Borrar Empresa</string>
                      </property>
                      <property name="toolTip" stdset="0" >
                        <string>Borra por completo una empresa</string>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
              <item>
                <widget class="QPushButton" name="pushButton32" >
                  <property name="text" >
                    <string>&amp;Importar / Exportar Datos</string>
                  </property>
                  <property name="shortcut" >
                    <string>Alt+I</string>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer8" >
                  <property name="sizeHint" >
                    <size>
                      <width>21</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Vertical</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </widget>
          <widget class="QWidget" name="TabPage" >
            <attribute name="title" >
              <string>Bulma&amp;Fact</string>
            </attribute>
            <layout class="QVBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <item>
                <layout class="QHBoxLayout" >
                  <property name="margin" >
                    <number>0</number>
                  </property>
                  <item>
                    <widget class="QLineEdit" name="lineEditA_1_2" >
                      <property name="enabled" >
                        <bool>true</bool>
                      </property>
                      <property name="text" >
                        <string>Nom Empresa</string>
                      </property>
                      <property name="readOnly" >
                        <bool>true</bool>
                      </property>
                      <property name="toolTip" stdset="0" >
                        <string>Nombre de la empresa</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QPushButton" name="pushButtonA_6_2" >
                      <property name="text" >
                        <string/>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <spacer name="spacer2_2" >
                      <property name="sizeHint" >
                        <size>
                          <width>211</width>
                          <height>21</height>
                        </size>
                      </property>
                      <property name="sizeType" >
                        <enum>Expanding</enum>
                      </property>
                      <property name="orientation" >
                        <enum>Horizontal</enum>
                      </property>
                    </spacer>
                  </item>
                </layout>
              </item>
              <item>
                <layout class="QHBoxLayout" >
                  <property name="margin" >
                    <number>0</number>
                  </property>
                  <item>
                    <widget class="QPushButton" name="pushButtonA_61_2" >
                      <property name="text" >
                        <string>&amp;Clonar Empresa</string>
                      </property>
                      <property name="toolTip" stdset="0" >
                        <string>Crea una nueva empresa identica a otra existente</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QPushButton" name="m_backup_2" >
                      <property name="text" >
                        <string>Copia &amp;de Seguridad
 (BACKUP)</string>
                      </property>
                      <property name="toolTip" stdset="0" >
                        <string>Guarda una copia de seguridad de una empresa</string>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
              <item>
                <layout class="QHBoxLayout" >
                  <property name="margin" >
                    <number>0</number>
                  </property>
                  <item>
                    <widget class="QPushButton" name="pushButtonA_1_2" >
                      <property name="text" >
                        <string>Crear &amp;Nueva Empresa</string>
                      </property>
                      <property name="toolTip" stdset="0" >
                        <string>Crea una empresa nueva sin datos</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QPushButton" name="m_borrarempresa_2" >
                      <property name="text" >
                        <string>&amp;Borrar Empresa</string>
                      </property>
                      <property name="toolTip" stdset="0" >
                        <string>Borra por completo una empresa</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QPushButton" name="pushButtonA_4_2" >
                      <property name="text" >
                        <string>Res&amp;taurar copia
 (RESTORE)</string>
                      </property>
                      <property name="toolTip" stdset="0" >
                        <string>Restaura la copia de seguridad de una empresa</string>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
              <item>
                <widget class="QPushButton" name="pushButton32_2" >
                  <property name="text" >
                    <string>&amp;Importar / Exportar Datos</string>
                  </property>
                  <property name="shortcut" >
                    <string>Alt+I</string>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer7_2" >
                  <property name="sizeHint" >
                    <size>
                      <width>81</width>
                      <height>60</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Vertical</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </widget>
          <widget class="QWidget" name="TabPage" >
            <attribute name="title" >
              <string>Parametros</string>
            </attribute>
            <layout class="QGridLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <item row="0" column="0" >
                <widget class="Q3Table" name="m_tconfiguracion" >
                  <property name="mouseTracking" >
                    <bool>true</bool>
                  </property>
                  <property name="frameShadow" >
                    <enum>Sunken</enum>
                  </property>
                  <property name="resizePolicy" >
                    <enum>Default</enum>
                  </property>
                  <property name="dragAutoScroll" >
                    <bool>true</bool>
                  </property>
                  <property name="numRows" >
                    <number>3</number>
                  </property>
                  <property name="numCols" >
                    <number>3</number>
                  </property>
                  <property name="showGrid" >
                    <bool>true</bool>
                  </property>
                  <property name="selectionMode" >
                    <enum>SingleRow</enum>
                  </property>
                  <column>
                    <property name="text" >
                      <string>Nombre</string>
                    </property>
                  </column>
                  <column>
                    <property name="text" >
                      <string>Valor</string>
                    </property>
                  </column>
                  <column>
                    <property name="text" >
                      <string>idconfiguracion</string>
                    </property>
                  </column>
                </widget>
              </item>
            </layout>
          </widget>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <spacer name="spacer1" >
              <property name="sizeHint" >
                <size>
                  <width>431</width>
                  <height>31</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="pushButtonF_X" >
              <property name="text" >
                <string>Sal&amp;ir</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>pushButtonA_5</tabstop>
    <tabstop>pushButtonA_1</tabstop>
    <tabstop>m_borrarempresa</tabstop>
    <tabstop>m_backup</tabstop>
    <tabstop>pushButtonA_4</tabstop>
    <tabstop>m_tab</tabstop>
    <tabstop>pushButtonF_X</tabstop>
    <tabstop>lineEditA_1</tabstop>
  </tabstops>
</ui>
